### DISCUSSION TITLE
Problem with PylonCapture and CsvWriter

### DISCUSSION THREAD

**Discussion by AdrianDuszkiewicz** [bonsai-rx/bonsai#1272] [Q&A]
Hi everyone! I'm using Bonsai to track and record video from a Basler camera. With PylonCapture 0.1.0 and Pylon 5.1 everything runs flawlessly but after I updated everything (PylonCapture, 0.3.0, Pylon  7.1, Bonsai 2.7.2), the tracking part is still totally fine (including CsvWriter) but VideoWriter doesn't save the video file to the hard drive. The workflow is exactly the same in both cases and the problem persists on two separate PC's.

 I'm probably doing something wrong but can't figure out what... I'd be very thankful for any tips! 

Workflow screenshot below 

![Untitled](https://user-images.githubusercontent.com/55764474/223213324-5a5a32ad-a840-489d-a75b-a0ed3f75051f.png)

**Comment by bruno-f-cruz** [comment]
Hey 
Can you include the exact workflow you are using?
Just to make sure, are you using Bonsai64 or 32?
Cheers,
Bruno

**Reply by AdrianDuszkiewicz** [reply]
Hi Bruno,thank you for responding! I'm using Bonsai 64. 

Workflow below
[Bonsai_2Box_Feb2023.bonsai.zip](https://github.com/bonsai-rx/bonsai/files/10902705/Bonsai_2Box_Feb2023.bonsai.zip)

**Reply by bruno-f-cruz** [reply]
Make sure you add the extension to your file name (e.g. ".avi")

**Reply by AdrianDuszkiewicz** [reply]
It worked! Thank you very much!
